apps/src/blockly/addons/cdoSerializationHelpers.ts

```diff
@@ -405,13 +429,21 @@ export function convertFunctionsXmlToJson(functionsXml) {
  * @param {Object} projectState - The saved project in JSON (blocks and procedures).
  * @param {Object} proceduresState - The shared procedures in JSON (blocks and procedures).
  * @returns {Object} - The updated project state with shared procedures appended.
+ * TODO: define a type for projectState and proceduresState. Blockly defines these as any.
  */
-export function appendProceduresToState(projectState, proceduresState) {
+export function appendProceduresToState(
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  projectState: any,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  proceduresState: any
```

Comment:
I assume 'projectState` and `proceduresState` are some kind of complex nested objects where we can't predict what exactly will be in there?

Reply:
Kind of, I am actually working on a follow-up that defines these precisely. I just needed to put `any` here to make progress.